modules/dmrpp_module/build_dmrpp_util.cc

```diff
@@ -2163,17 +2163,18 @@ void qc_input_file(const string &file_fqn)
         throw BESInternalFatalError(msg.str(), __FILE__, __LINE__);
     }
 
-    //HDF5 and NetCDF3 signatures:
-    const char hdf5Signature[] = {'\211', 'H', 'D', 'F', '\r', '\n', '\032', '\n'};
+    //NetCDF3 signatures:
     const char netcdf3Signature[] = {'C', 'D', 'F'};
 
     //Read the first 8 bytes (file signature) from the file
     string signature;
     signature.resize(8);
     file.read(&signature[0], signature.size());
 
+    htri_t temp_is_hdf5 = H5Fis_hdf5(file_fqn.c_str());
+    
+    bool isHDF5 = (temp_is_hdf5>0)?true:false;
```

Comment:
Using the ternary operator is fine, but you can also say `bool isHDF5 = (temp_is_hdf5>0);`